### PR TITLE
[docs] Describe the debugOnFail option

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs

### CODE REVIEW

File: docs/articles/documentation/using-testcafe/programming-interface/runner.md [hunk 1]
```diff
@@ -368,7 +368,8 @@ Parameter         | Type    | Description
...
 `speed`           | Number  | Specifies the speed of test execution. Should be a number between `1` (the fastest) and `0.01` (the slowest). If speed is also specified for an [individual action](../../test-api/actions/action-options.md#basic-action-options), the action speed setting overrides test speed. | `1`
-`debugMode`       | Boolean | Specifies if tests run in the debug mode. If this option is enabled, test execution is paused before the first action or assertion allowing you to invoke the developer tools and debug. | `false`
+`debugMode`       | Boolean | Specifies if tests run in the debug mode. If this option is enabled, test execution is paused before the first action or assertion allowing you to invoke the developer tools and debug. In the debug mode, you can execute the test step-by-step to reproduce its incorrect behavior. | `false`
+`debugOnFail`     | Boolean | Specifies whether to enter the debug mode when a test fails. If enabled, the test is paused at the moment it fails so that you can explore the tested page to determine the cause of the fail. During this time, you can interact with the web page as if it was regularly opened in a browser. | `false`
```
Comment (kirovboris) [inline docs/articles/documentation/using-testcafe/programming-interface/runner.md]: Missed comma before "so that you..."

**Comment by AlexanderMoskovkin** [comment]
The **Unlock page** switch is not implemented yet, so let's don't merge it for now

**Comment by VasilyStrelyaev** [comment]
I think it's about time to review, isn't it?

**Comment by AlexanderMoskovkin** [comment]
> The Unlock page switch is not implemented yet, so let's don't merge it for now

I'm working on this right now

**Pull request merged**